Go 1.15: conversion from int to string yields a string of one rune

Go 1.15 rc 1 on Fedora Rawhide:

```
Testing    in: /builddir/build/BUILD/otto-c382bd3c16ff2fef9b5fe0dd8bf4c4ec6bfe62c1/_build/src
         PATH: /builddir/build/BUILD/otto-c382bd3c16ff2fef9b5fe0dd8bf4c4ec6bfe62c1/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/otto-c382bd3c16ff2fef9b5fe0dd8bf4c4ec6bfe62c1/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/robertkrimen/otto/version=0 -X github.com/robertkrimen/otto/version.commit=c382bd3c16ff2fef9b5fe0dd8bf4c4ec6bfe62c1 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/robertkrimen/otto
github.com/robertkrimen/otto
# github.com/robertkrimen/otto
./builtin.go:51:16: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
./builtin.go:53:16: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
./builtin.go:53:35: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
./regexp_test.go:259:15: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
FAIL	github.com/robertkrimen/otto [build failed]
```


See https://github.com/golang/go/issues/32479

--- a/builtin.go
+++ b/builtin.go
@@ -48,9 +48,9 @@ var parseInt_alphabetTable = func() []string {
 	for radix := 3; radix <= 36; radix += 1 {
 		alphabet := table[radix-1]
 		if radix <= 10 {
-			alphabet += string(radix + 47)
+			alphabet += string(rune(radix + 47))
 		} else {
-			alphabet += string(radix+54) + string(radix+86)
+			alphabet += string(rune(radix+54)) + string(rune(radix+86))
 		}
 		table = append(table, alphabet)
 	}

--- a/regexp_test.go
+++ b/regexp_test.go
@@ -256,7 +256,7 @@ func TestRegExp_controlCharacter(t *testing.T) {
 		test, _ := test()
 
 		for code := 0x41; code < 0x5a; code++ {
-			string_ := string(code - 64)
+			string_ := string(rune(code - 64))
 			test(fmt.Sprintf(`
                 var code = 0x%x;
                 var string = String.fromCharCode(code %% 32);

